s6/debian-root/etc/services.d/lighttpd/run

```diff
@@ -2,11 +2,23 @@
 
 s6-echo "Starting lighttpd"
 
-# Touch log files to ensure they exist (create if non-existing, preserve if existing)
-touch /var/log/lighttpd/access.log /var/log/lighttpd/error.log
+if [[ 1 -eq ${WEBLOGS_STDOUT:-0} ]]; then
+  #lighthttpd cannot use /dev/stdout https://redmine.lighttpd.net/issues/2731
+  rm -f /var/log/lighttpd/access.log
```

Comment:
I wonder if it is also worth putting a check in here to only rm the file if it is not a pipe, i.e `if [[ ! -p /var/log/lighttpd/access.log ]] remove it` (and same for the error log)

Might be overkill!